src/components/HTMLEngineProvider/HTMLRenderers/PreRenderer.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {ScrollView} from 'react-native-gesture-handler';
+import {View} from 'react-native';
+import _ from 'underscore';
+import withLocalize from '../../withLocalize';
+import htmlRendererPropTypes from './htmlRendererPropTypes';
+
+const PreRenderer = (props) => {
+    const TDefaultRenderer = props.TDefaultRenderer;
+    const defaultRendererProps = _.omit(props, ['TDefaultRenderer']);
+
+    return (
+        <ScrollView horizontal>
```

Comment:
Not able to scroll via touchpad two-finger scrolling.

Reply:
> Not able to scroll via touchpad two-finger scrolling.

@parasharrajat The reason for this issue is inverted scroll on InvertedFlatList component.

In InvertedFlatList this function is creating error because here we make scroll manual and force to stop other scrolling operations with preventDefault()

```js
  invertedWheelEvent(e) {
        this.list.getScrollableNode().scrollTop -= e.deltaY;
        e.preventDefault();
    }
```

I did some work on this issue but the only solution is remove the manual scroll operation from InvertedFlatList component.

### Disabled the inverted scroll

https://user-images.githubusercontent.com/61627150/166339501-b786325d-d0c0-4f18-9108-091ac8a9f51f.mov

Reply:
I would not call that the only solution. Please take some time and try to come up with some approaches and we can decide the best.

Reply:
> I would not call that the only solution. Please take some time and try to come up with some approaches and we can decide the best.

@parasharrajat Okay, I'll spend more time, but it is absolutely necessary to remove preventDefault from that function.
Now i am working on how we can proceed for inverted scroll without preventDefault

Reply:
@parasharrajat I did some research and realized that a similar issue has been encountered on the platform before. 
[Issue Link](https://github.com/Expensify/App/issues/4132)
When I look at the solution of this problem, if it is in the edit box, it override the parent wheel listener in the list. When I saw that this is an approved solution, I developed a solution similar to this and if our code block is have a scrollable length then i removing the parent listener when trying to scroll this code block.

### Changes

Native and web codes are separated. Nothing has been changed in native, a listener has been added to the code block on the web side.

PR Ready.

### Simulation Videos

#### native app

https://user-images.githubusercontent.com/61627150/166453522-394c150c-5e2e-41af-9646-140ffadeb9a2.mp4

#### web app

https://user-images.githubusercontent.com/61627150/166453849-6817064f-817a-4ef1-8369-d33368749dbf.mp4